plugin/storage/integration/cassandra_test.go

```diff
@@ -64,24 +66,27 @@ func (s *CassandraStorageIntegration) initializeCassandra(t *testing.T) {
 		"--cassandra.password=" + password,
 		"--cassandra.username=" + username,
 		"--cassandra.keyspace=jaeger_v1_dc1",
+	}, cassandra.NewFactory)
+	af := s.initializeCassandraFactory(t, []string{
 		"--cassandra-archive.keyspace=jaeger_v1_dc1_archive",
 		"--cassandra-archive.enabled=true",
 		"--cassandra-archive.servers=127.0.0.1",
 		"--cassandra-archive.basic.allowed-authenticators=org.apache.cassandra.auth.PasswordAuthenticator",
 		"--cassandra-archive.password=" + password,
 		"--cassandra-archive.username=" + username,
```

Comment:
could we drop these? The inheritance should take care of them.

Reply:
One way we could achieve that is if we used meta-factory here instead of explicit `C*` factory. It would still be validated to be `C*` because of the flags we pass.

Reply:
Maybe in the next PR?